cmd/security-secretstore-setup/entrypoint.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/dumb-init /bin/sh
+#  ----------------------------------------------------------------------------------
+#  Copyright (c) 2019 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+#  SPDX-License-Identifier: Apache-2.0'
+#  ----------------------------------------------------------------------------------
+
+set -e
+
+echo "creating /vault/config/assets"
+
+# create token directory and
+# grant permissions of folders for vault:vault
+mkdir -p /vault/config/assets
+chown -R 100:1000 /vault/
```

Comment:
```suggestion
chown -Rh 100:1000 /vault/
```

Been burned too many times by forgetting `-h` that I can't help but suggest it.

Reply:
good catch